debian/control.in

```diff
@@ -0,0 +1,10 @@
+Package: ckan
+Version: @VERSION@
+Architecture: all
+Section: utils
+Priority: optional
+Depends: mono-runtime (>=4.0), ca-certificates-mono, libmono-microsoft-csharp4.0-cil, liblog4net1.2-cil, libnewtonsoft-json5.0-cil
+Maintainer: Michele Ippolito <m.ippolito@outlook.com>
+Description: Official KSP-CKAN client.
+ Official client for the Comprehensive Kerbal Archive Network.
```

Comment:
And add the acronym (CKAN) after the spelled out name, so that we're being perfectly obvious where the name comes from.

Reply:
OK.